openfisca_us/variables/finance/benefit/school_meals.py

```diff
@@ -0,0 +1,43 @@
+from openfisca_core.model_api import *
+from openfisca_us.entities import *
+from openfisca_us.tools.general import *
+from openfisca_us.variables.demographic.spm_unit import *
+
+# add new variable for enum tier
+
+class school_meal_subsidy(Variable):
+    value_type = float
+    entity = SPMUnit
+    definition_period = YEAR
+    documentation = ""
+
+    def formula(spm_unit, period, parameters):
+        # Get state and poverty ratio for SPM unit.
+        state_group = spm_unit("spm_unit_state_group", period)
+        poverty_ratio = spm_unit("poverty_ratio", period)
+        # Get parameters.
+        p_school_meals = parameters(period).benefit.school_meals
+        p_income_limit = p_school_meals.income_limit
+        # Look up the free/reduced/full subsidy tier for each SPM unit by
+        # poverty ratio.
+        tier = where(
+            poverty_ratio < p_income_limit["free"],
```

Comment:
I might suggest a `select` (alias for `np.select`) statement here to make it more readable:
```python
tier = select([
    poverty_ratio < p_income_limit.free,
    poverty_ratio < p_income_limit.reduced_price,
    poverty_ratio >= p_income_limit.reduced_price,
], ["free", "reduced_price", "paid"])